packages/image-block/src/components/helpers.ts

```diff
@@ -74,6 +74,19 @@ export const getCaptionPlugins = (appBridge: AppBridgeBlock) =>
             new ResetFormattingPlugin(),
         ]);
 
+export const getTotalImagePadding = (blockSettings: Settings): CSSProperties => {
```

Comment:
Add a unit test for that function

Reply:
We don't have that package within this block. I just added 3 tests to the ImageBlock.spec.ct.tsx (and fixed already a bug :D)

Reply:
For me this is also fine